packages/kolibri-tools/lib/i18nSyncContext.js

```diff
@@ -0,0 +1,343 @@
+// Import packages
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const recast = require('recast');
+const traverse = require('ast-traverse');
+const get = require('lodash/get');
+const parseCsvSync = require('csv-parse/lib/sync');
+const vueCompiler = require('vue-template-compiler');
+const logging = require('./logging');
+
+// Glob path patterns
+// All JS files not in node_modules
+const JS_GLOB = path.resolve('./kolibri') + '/!(node_modules)/**/*.js';
+// All Vue files not in node_modules
+const VUE_GLOB = path.resolve('./kolibri') + '/!(node_modules)/**/*.vue';
+// All of the files downloaded from Crowdin
+const CSV_PATH = path.resolve('./kolibri/locale/CSV_FILES/');
+
+// -------------------- //
+// Processing Functions //
+// -------------------- //
+
+const processVueFiles = (files, definitions) => {
```

Comment:
is there a particular reason we're defining functions in here as 

```js
const funcName = (args) => {}
```

instead of 

```js
function funcName(args) {}
```

if we have a choice, I think the latter is much more clear

Reply:
@indirectlylit sorry about that - honestly it is just a habit. I like the arrow function format. I've updated all functions defined at the root level with standard JS function declarations.

However, for callbacks passed to various methods (ie, `[1,2,3].map(i => ...)`) I have kept the functions defined as arrow functions.